src/ResourceManager/IotHub/Commands.IotHub/Properties/Resources.resx

```diff
@@ -138,10 +141,16 @@
   <data name="RemoveIotHub" xml:space="preserve">
     <value>Remove IoT Hub</value>
   </data>
+  <data name="RemoveIotHubCertificate" xml:space="preserve">
+    <value>Remove IoT Hub Certificate</value>
+  </data>
   <data name="RemoveIotHubKey" xml:space="preserve">
     <value>Remove IoT Hub Key</value>
   </data>
   <data name="UpdateIotHub" xml:space="preserve">
     <value>Update IoT Hub </value>
   </data>
+  <data name="VerifyIotHubCertificate" xml:space="preserve">
+    <value>Add an Iot Hub Verified Certificate</value>
```

Comment:
Maybe this should read: "Verify an Iot Hub Certificate"

Reply:
Done